src/gui/georeferencing_dialog.cpp

```diff
@@ -251,7 +251,7 @@ GeoreferencingDialog::GeoreferencingDialog(
 	edit_layout->addRow(tr("Map coordinates:"), map_ref_layout);
 	edit_layout->addRow(projected_ref_label, projected_ref_layout);
 	edit_layout->addRow(tr("Geographic coordinates:"), geographic_ref_layout);
-	edit_layout->addRow(show_refpoint_label, link_label);
+	edit_layout->addItem(Util::SpacerItem::create(this));
```

Comment:
I would prefer to not add a spacer.

Reply:
I agree, I compared the look of both variants and also prefer the one without the SpacerItem.
In addition I added a commit that fixes an annoying warning due to comparing int and unsigned int.